client/reducers/socialClubReducers.js

```diff
@@ -27,7 +27,13 @@ export function socialClubs(state = initialState.socialClubs, action) {
 export function socialClub(state = initialState.socialClub, action) {
   switch(action.type){
     case GET_CLUB:
-      return action.club;
+      return action.payload;
+    case CREATE_EVENT:
+      return Object.assign({}, state, {
```

Comment:
Can you double check this line here? It's just merging state into state again, a better way to put it might be:
```
Object.assign({}, state, action.payload)
```

Reply:
`action.payload` contains only a new event. However, the state passed into that reducer is the entire socialClub. Hence, we only want to change the events key in the socialClub object by adding the new event to the list of events.